datafusion/core/src/lib.rs

```diff
@@ -161,11 +161,17 @@
 //! * Projection: [`ProjectionExec`](physical_plan::projection::ProjectionExec)
 //! * Filter: [`FilterExec`](physical_plan::filter::FilterExec)
 //! * Grouped and non-grouped aggregations: [`AggregateExec`](physical_plan::aggregates::AggregateExec)
+//! * Hash Join: [`HashJoinExec`](physical_plan::hash_join::HashJoinExec)
+//! * Cross Join: [`HashJoinExec`](physical_plan::cross_join::CrossJoinExec)
+//! * Sort Merge Join: [`SortExec`](physical_plan::sort_merge_join::SortMergeJoinExec)
```

Comment:
```suggestion
//! * Sort Merge Join: [`SortMergeJoinExec`](physical_plan::sort_merge_join::SortMergeJoinExec)
```

Reply:
Wow, I really messed this up. Sorry for the extra work.

Reply:
No apologies needed -- Thank you for improving the docs. I am just glad a review offered some value 😆